File: Binary.java
import java.util.Scanner;
class Binary{
    public static void main(String []args){
        System.out.println("enter the size of an array:");
        Scanner sc=new Scanner(System.in);
        int a[];
        int n=sc.nextInt();
        a= new int[n];
        for(int i=0;i<n;i++){
            System.out.println("enter the element:");
            a[i]=sc.nextInt();
        }
        for(int i=0;i<n;i++){
            System.out.println(a[i]);
        }
    }
}

File: Binary.py
def Binary(lst,key):
    l=0
    h=len(lst)-1
    while l<=h:
        mid=(l+h)//2
        if lst[mid]==key:
            print("element found at ",mid)
            break
        elif lst[mid]>mid:
            l=mid+1
        else:
            h=mid-1
    if l>h:
        print("element not found")
l=[]
for i in range(int(input("enter the size of an array:"))):
    l.append(int(input("enter the element :")))
Binary(l,int(input("enter the element to be found")))